src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -58,23 +54,46 @@ public DriveTrain() {
     m_leftFront.setInverted(false);
     m_leftBack.setInverted(false);
 
+    m_leftBack.getEncoder(EncoderType.kQuadrature, Constants.DriveTrain.ENCODER_RESOLUTION)
+        .setPositionConversionFactor(Constants.DriveTrain.ENCODER_METERS_PER_TICK);
+    m_rightBack.getEncoder(EncoderType.kQuadrature, Constants.DriveTrain.ENCODER_RESOLUTION)
+        .setPositionConversionFactor(Constants.DriveTrain.ENCODER_METERS_PER_TICK);
+
+    m_leftBack.getEncoder(EncoderType.kQuadrature, Constants.DriveTrain.ENCODER_RESOLUTION)
+        .setVelocityConversionFactor(Constants.DriveTrain.ENCODER_MPS_PER_RPM);
+    m_rightBack.getEncoder(EncoderType.kQuadrature, Constants.DriveTrain.ENCODER_RESOLUTION)
+        .setVelocityConversionFactor(Constants.DriveTrain.ENCODER_MPS_PER_RPM);
```

Comment:
Create a `CANEncoder` rather than repeating this.

Also, comment out this code for the external encoder, and use the built-in for now since the external are not wired yet.

Reply:
resolved in 93bb0d3